contracts/PvpArena.sol

```diff
@@ -35,54 +41,119 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     int128 public extraCostPerLevel;
 
     /// @dev fighters available by tier (1-10, 11-20, etc...)
-    mapping (uint8 => Fighter[]) private fightersByTier;
+    mapping(uint8 => Fighter[]) private fightersByTier;
     /// @dev fighters by player address
-    mapping (address => Fighter[]) private fightersByPlayer;
+    mapping(address => Fighter[]) private fightersByPlayer;
+
+    ///@dev mapping to track the characters used
+    mapping(uint256 => bool) public isCharacterUsed;
```

Comment:
In most languages the `is` prefix is used to name booleans. Since this is a mapping and therefore there are multiple   characters here, I'd suggest a name such as `usedCharacters`